packages/paste-core/components/form/stories/index.stories.tsx

```diff
@@ -0,0 +1,425 @@
+  FormSectionHeading,
+} from '../src';
+
+// eslint-disable-next-line import/no-default-export
+export default {
+  title: 'Components/Form',
+  component: Form,
+  subcomponents: {
+    FormActions,
+    FormControl,
+    FormControlTwoColumn,
+    FormSection,
+    FormSectionDescription,
+    FormSectionHeading,
+  },
+};
+
+const countriesList = [
+  'Bangladesh',
+  'Brazil',
+  'China',
+  'Ethiopia',
+  'India',
+  'Indonesia',
+  'Japan',
+  'Mexico',
+  'Nigeria',
+  'Pakistan',
+  'Philippines',
+  'United States',
+];
+
+const statesList = [
+  {value: 'AK', name: 'Alaska'},
+  {value: 'AL', name: 'Alabama'},
+  {value: 'AZ', name: 'Arizona'},
+  {value: 'CA', name: 'California'},
+  {value: 'CO', name: 'Colorado'},
+  {value: 'CT', name: 'Connecticut'},
+  {value: 'ID', name: 'Idaho'},
+  {value: 'IL', name: 'Illinois'},
+  {value: 'IN', name: 'Indiana'},
+  {value: 'KS', name: 'Kansas'},
+  {value: 'KY', name: 'Kentucky'},
+  {value: 'LA', name: 'Louisiana'},
+  {value: 'MN', name: 'Minnesota'},
+  {value: 'MO', name: 'Missouri'},
+  {value: 'MS', name: 'Mississippi'},
+  {value: 'MT', name: 'Montana'},
+];
+
+const componentsList = [
+  'Alert',
+  'Anchor',
+  'Button',
+  'Card',
+  'Heading',
+  'List',
+  'Modal',
+  'Paragraph',
+  'Popover',
+  'Tooltip',
+];
+
+function getMultiselectComboboxFilteredItems(inputValue: string): string[] {
+  const lowerCasedInputValue = inputValue.toLowerCase();
+
+  return componentsList.filter(function filterItems(item) {
+    return item.toLowerCase().includes(lowerCasedInputValue);
+  });
+}
+
+export const Default: StoryFn = () => {
+  const seed = useUIDSeed();
+  const [comboboxInputValue, setComboboxInputValue] = React.useState('');
+  const filteredItems = React.useMemo(
+    () => getMultiselectComboboxFilteredItems(comboboxInputValue),
+    [comboboxInputValue]
+  );
+
+  return (
+    <Form aria-labelledby={seed('form-heading')}>
+      <Heading id={seed('form-heading')} as="h2" variant="heading20" marginBottom="space0">
+        {/* Added UID to the heading because forms are HTML Landmarks. Axe was failing because in the stacked view, there were 4 forms with the same name and landmarks have to have unique names. */}
+        Profile settings {useUID()}
+      </Heading>
+      <Paragraph marginBottom="space0">Please add information to your Twilio profile.</Paragraph>
+      <FormSection>
+        <FormSectionHeading>Contact information</FormSectionHeading>
+        <FormSectionDescription>Please add the details of your contact information.</FormSectionDescription>
+        <FormControl>
+          <Label htmlFor={seed('phone-number')}>Phone number</Label>
+          <Input id={seed('phone-number')} type="tel" insertAfter={<InformationIcon decorative />} />
+        </FormControl>
+        <FormControl>
+          <Label htmlFor={seed('email')}>Email address</Label>
+          <Input id={seed('email')} type="email" />
+        </FormControl>
+        <MultiselectCombobox
```

Comment:
question: to clarify, you don't need to use `FormControl` with `MultiselectCombobox` bc it controls its own label, is that correct? What would happen if there _were_ a `FormControl` around it? (i'm thinking nothing, but wanted to check).